x/ibc/07-tendermint/types/misbehaviour.go

```diff
@@ -91,13 +91,13 @@ func (misbehaviour Misbehaviour) ValidateBasic() error {
 	}
 
 	// ValidateBasic on both validators
-	if err := misbehaviour.Header1.ValidateBasic(misbehaviour.ChainId); err != nil {
+	if err := misbehaviour.Header1.ValidateBasic(); err != nil {
```

Comment:
Add checks that `misbehaviour.Header1.ChainID == misbehaviour.ChainId` and `misbehaviour.Header2.ChainID == misbehaviour.ChainId`